### ISSUE TITLE
`file_missing` exceptions have been encountered by ingest, and during smoketest

### ISSUE THREAD

**Issue by xbrianh** [HumanCellAtlas/data-store#1611]
ingest exception (21-September, 2018): 
```2018-09-20 22:31:20,744  - ingest.api.dssapi - ERROR in dssapi.py:82 put_bundle(): Error in hca_client.put_bundle method call with params: {"uuid": "d6084590-a460-4957-a509-ba84e458a937", "version": "201
8-09-20T223105.508251Z", "replica": "aws", "files": [{"indexed": true, "name": "cell_suspension_0.json", "uuid": "13edd68a-4757-414c-8206-2c30c8b2c857", "content-type": "\"metadata/biomaterial\"", "versio
n": "2018-09-20T223058.476243Z"}, {"indexed": true, "name": "specimen_from_organism_0.json", "uuid": "60c7f068-1f85-4b0f-9c8c-354ac84e18d4", "content-type": "\"metadata/biomaterial\"", "version": "2018-09
-20T223058.904901Z"}, {"indexed": true, "name": "donor_organism_0.json", "uuid": "64b30d02-f080-4b3e-984d-0951f366e72f", "content-type": "\"metadata/biomaterial\"", "version": "2018-09-20T223059.282617Z"}
, {"indexed": true, "name": "sequence_file_0.json", "uuid": "2be43212-de25-4984-9222-f8ddb2f45496", "content-type": "\"metadata/file\"", "version": "2018-09-20T223059.424743Z"}, {"indexed": true, "name":
"sequence_file_1.json", "uuid": "36915d0d-60ba-482f-b88a-934b71a246f8", "content-type": "\"metadata/file\"", "version": "2018-09-20T223100.390874Z"}, {"indexed": true, "name": "project_0.json", "uuid": "7
5bca353-52d1-4fec-be31-9a3243d59430", "content-type": "\"metadata/project\"", "version": "2018-09-20T223100.608702Z"}, {"indexed": true, "name": "library_preparation_protocol_0.json", "uuid": "7701cdb1-70
f5-4a16-9e60-ed63294dfb13", "content-type": "\"metadata/protocol\"", "version": "2018-09-20T223100.763455Z"}, {"indexed": true, "name": "sequencing_protocol_0.json", "uuid": "02c0c01a-fe13-4c55-8113-33523ea3f7bb", "content-type": "\"metadata/protocol\"", "version": "2018-09-20T223100.951683Z"}, {"indexed": true, "name": "dissociation_protocol_0.json", "uuid": "29014757-ba20-47e6-a000-8644e33bfddc", "content-type": "\"metadata/protocol\"", "version": "2018-09-20T223101.130718Z"}, {"indexed": true, "name": "process_0.json", "uuid": "cb6b5052-6902-40cc-8b1c-e84a8c3a4323", "content-type": "\"metadata/process\"", "version": "2018-09-20T223101.302624Z"}, {"indexed": true, "name": "process_1.json", "uuid": "1412d2ce-d516-4a69-a287-c9ead247f41f", "content-type": "\"metadata/process\"", "version": "2018-09-20T223101.543838Z"}, {"indexed": true, "name": "process_2.json", "uuid": "cd90e108-09b5-49ac-a4f4-019f13736ff5", "content-type": "\"metadata/process\"", "version": "2018-09-20T223101.815368Z"}, {"indexed": true, "name": "links.json", "uuid": "b87be695-b7b5-4330-b25e-421668888840", "content-type": "\"metadata/links\"", "version": "2018-09-20T223102.043108Z"}, {"indexed": false, "name": "SRR3934496_1.fastq.gz", "uuid": "d4e8de62-d102-477d-bc6c-dfa4140d3545", "content-type": "data", "version": "2018-09-20T223102.476624Z"}, {"indexed": false, "name": "SRR3934496_2.fastq.gz", "uuid": "3568946b-91ec-4a16-b975-23f5ab16df95", "content-type": "data", "version": "2018-09-20T223105.275770Z"}], "creator_uid": 8008}
 2018-09-20 22:31:20,744  - __main__ - ERROR in export-to-dss.py:47 on_message(): An error occurred while putting bundle in DSS: file_missing: Could not find file 3568946b-91ec-4a16-b975-23f5ab16df95/2018-09-20T223105.275770Z. (HTTP 400). Details:
Traceback (most recent call last):
  File "/var/task/chalicelib/dss/error.py", line 54, in wrapper
    return func(*args, **kwargs)
  File "/var/task/chalicelib/dss/api/bundles/__init__.py", line 190, in put
    f"Could not find file {missing_file_user_metadata['uuid']}/{missing_file_user_metadata['version']}."
dss.error.DSSException
Traceback (most recent call last):
  File "/app/src/hca-ingest/ingest/exporter/ingestexportservice.py", line 370, in put_bundle_in_dss
    created_bundle = self.dss_api.put_bundle(bundle_uuid, created_files)
  File "/app/src/hca-ingest/ingest/api/dssapi.py", line 71, in put_bundle
    creator_uid=self.creator_uid
  File "/usr/local/lib/python3.7/site-packages/hca/util/__init__.py", line 155, in __call__
    return self._consume_response(self._request(kwargs))
  File "/usr/local/lib/python3.7/site-packages/hca/util/__init__.py", line 143, in _request
    raise SwaggerAPIException(response=res)
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 3568946b-91ec-4a16-b975-23f5ab16df95/2018-09-20T223105.275770Z. (HTTP 400). Details:
Traceback (most recent call last):
  File "/var/task/chalicelib/dss/error.py", line 54, in wrapper
    return func(*args, **kwargs)
  File "/var/task/chalicelib/dss/api/bundles/__init__.py", line 190, in put
    f"Could not find file {missing_file_user_metadata['uuid']}/{missing_file_user_metadata['version']}."
dss.error.DSSException
```

smoketest (27-September, 2018):
https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/data-store/-/jobs/8621
```  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/bin/hca", line 11, in <module>
    sys.exit(main())
  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/lib/python3.6/site-packages/hca/cli.py", line 75, in main
    result = parsed_args.entry_point(parsed_args)
  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/lib/python3.6/site-packages/hca/util/__init__.py", line 479, in arg_forwarder
    return command(**command_args)
  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/lib/python3.6/site-packages/hca/dss/__init__.py", line 321, in upload
    files=file_args)
  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/lib/python3.6/site-packages/hca/util/__init__.py", line 155, in __call__
    return self._consume_response(self._request(kwargs))
  File "/home/hca_cicd/data-store/smoketest-mg2shet8.tmp/venv/lib/python3.6/site-packages/hca/util/__init__.py", line 143, in _request
    raise SwaggerAPIException(response=res)
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file ff23c22f-08a2-44bb-a968-841a1968f118/2018-09-26T195117.367201Z. (HTTP 400). Details:
Traceback (most recent call last):
  File "/var/task/chalicelib/dss/error.py", line 55, in wrapper
    return func(*args, **kwargs)
  File "/var/task/chalicelib/dss/api/bundles/__init__.py", line 190, in put
    f"Could not find file {missing_file_user_metadata['uuid']}/{missing_file_user_metadata['version']}."
dss.error.DSSException
```

**Comment by parthshahva** [comment]
More examples. This is now causing 29 failures consistently in the 100 cell test.
```
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file fb72ae76-1a5f-494e-90e0-34567de8febf/2018-09-27T203352.952632Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file fb72ae76-1a5f-494e-90e0-34567de8febf/2018-09-27T203352.952632Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 588a54bd-7bbb-4ea8-a0b3-13ea4f0ca8c2/2018-09-27T203353.069893Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file ef67329c-d9dd-4e42-8f19-e45a93262b24/2018-09-27T203404.945585Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 3fbc4764-c878-4284-be3d-4dd33460b3bc/2018-09-27T203405.468546Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file b1325b7d-1368-4d9f-8cdd-10526e1a0b80/2018-09-27T203415.042155Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 30ecdbe5-a94b-4f0d-a0fb-b7875ed9bdb5/2018-09-27T203411.247228Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 994348ba-54bd-40ca-9003-962b499fab09/2018-09-27T203417.746440Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 795a2463-dbaa-4c8a-9488-3a8dd77ca549/2018-09-27T203419.125430Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file a9852e73-cfb3-4013-b055-4b5def9f7ee2/2018-09-27T203422.024002Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file f19c5b69-0270-4080-b7c4-a3ca274edbca/2018-09-27T203426.176403Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file c35a8285-1297-4d73-95ac-fdaa57294a5e/2018-09-27T203427.110964Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 7c9c5a2b-58e2-4b58-9661-f43ba07a00e3/2018-09-27T203429.636194Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file f4d77179-542c-4e62-9e70-f6b30e03fa5c/2018-09-27T203426.257291Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file a1d2dae8-239c-4560-b24b-12c351002074/2018-09-27T203447.673305Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file d374d9a5-16c1-4491-8447-4105b345d8d2/2018-09-27T205831.957005Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file e1a9b853-996b-4860-a146-bb6b7e26805a/2018-09-27T210120.127609Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 910c33ec-c056-4885-a280-f46db97d0c35/2018-09-27T210121.485356Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 93e30604-a6d5-4aa6-af58-a797d2325246/2018-09-27T210127.742303Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file df4aedac-57b6-4d50-a456-5e1e85155f44/2018-09-27T210131.785990Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 92a35797-739b-4e35-82be-06f7f87af5f5/2018-09-27T210132.241121Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 6a427589-bec8-4320-a533-37a9393a531d/2018-09-27T210143.295662Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file c6524042-2210-412c-8d3e-d65a99f6113d/2018-09-27T210155.618055Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 7ab9c2ad-3fd3-462d-9dfe-a541f89deeef/2018-09-27T210154.569709Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 1233658f-3c52-4985-ba54-721e1fe58423/2018-09-27T210200.370658Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file 03df8614-9e2e-4c7e-83ff-a3192227f0f0/2018-09-27T210341.285829Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file ed2c6fb4-51e5-4283-b681-1f94d8dddd8e/2018-09-27T210341.381373Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file d125a78d-29f4-4f27-8c5f-5bfdb8ebdaf7/2018-09-27T210411.177903Z. (HTTP 400). Details:\n
hca.util.exceptions.SwaggerAPIException: file_missing: Could not find file d39f9838-9ccf-4514-bea1-89863771b245/2018-09-27T210446.343729Z. (HTTP 400). Details:\n
```

**Comment by Bento007** [comment]
it looks like there is a failure in the subscription to the AWS bucket. Notification are not being sent to indexer, and sync is failing to replicate to GCP

**Comment by Bento007** [comment]
- We should add some status logs for ins messages.
- Sync is still failing to copy to GCP
- S3 SNS notifications are not triggering dss-indexer
- Need to understand what is going on in the step function to cause it to fail
- Need to figure out why the indexer is not being triggered.

**Comment by xbrianh** [comment]
#1632 will fix example from @rolando-ebi

**Comment by xbrianh** [comment]
Fixed by #1635
